src/main/java/bio/terra/service/tabulardata/LoadHistoryUtil.java

```diff
@@ -0,0 +1,16 @@
+package bio.terra.service.tabulardata;
+
+public class LoadHistoryUtil {
+
+  public static final String LOAD_TAG_FIELD_NAME = "load_tag";
```

Comment:
Why are these declared here instead of fields inside `StorageTableDao`? They are only used inside the class, and putting them here means that `StorageTableDao` needs to import them to use them.

Reply:
These are used in both `StorageTableDao` and `BigQueryPdao`

Reply:
Makes sense, I missed that